kahuna/public/js/leases/leases.js

```diff
@@ -71,21 +75,30 @@ leases.controller('LeasesCtrl', [
                     ctrl.newLease.notes = noteWithClause.join(', ');
                 }
 
-                let syndLeases = ctrl.leases.leases.filter((l) =>
+                const incomingLeaseIsSyndication = ctrl.access.endsWith('-syndication');
+                const syndLeases = ctrl.leases.leases.filter((l) =>
```

Comment:
minor: we could reuse [`isLeaseSyndication`](https://github.com/guardian/grid/pull/2437/files#diff-1260212b742426509a02c95e2a8f6332R137) to be DRYer?

Reply:
👍